extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -119,6 +119,12 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
     return null;
   }
 
+  /** @override */
+  getSigningServiceNames_() {
+    // TODO(levitzky) Add dev key name once it goes live.
+    return getMode().localDev ? ['google'] : ['google'];
```

Comment:
I agree with @lannka. I'm concerned with the case where the "google" signing server, for example, might change its URL, at which point each ad network that relies on that server for verification would have to manually update its implementations.